lib/monotonic_tick_count.rb

```diff
@@ -27,20 +27,20 @@ def inspect
     "monotonic tick count #{@tick_count_f}"
   end
 
-  # When the RHS is a duration, returns an offset from self
-  # When the RHS is a MonotonicTickCount, returns the difference between the two in fractional seconds
+  # When the RHS is a convertible to float, returns an offset to the current tick count
+  # When the RHS is a MonotonicTickCount,   returns the difference in float seconds
   def -(other)
-    if other.respond_to?(:from_now)
-      self + -other
-    elsif other.respond_to?(:tick_count_f)
+    if other.respond_to?(:tick_count_f)
       @tick_count_f - other.tick_count_f
+    elsif other.respond_to?(:to_f)
+      self + -other
     else
-      raise ArgumentError, "Other operand must be an ActiveSupport::Duration, #{self.class}, or equivalent"
+      raise ArgumentError, "Other operand must be another #{self.class} or respond to to_f"
     end
   end
 
   def +(other)
-    other.respond_to?(:from_now) or raise ArgumentError, "Other operand must be an ActiveSupport::Duration or equivalent"
+    other.respond_to?(:to_f) or raise ArgumentError, "Other operand must respond to to_f"
```

Comment:
Debated whether or not to use this, kind of dangerous since a lot of classes define a `to_f` method 
ex) "definitely not numeric".to_f => 0.0

But it seemed like it was reasonable to allow for any rhs that could be interpreted as float seconds to be valid. This also required less changes, than using whitelist set of classes [Duration, Numeric].

Reply:
Yeah, I don't like that `String` and `NilClass` both implement `to_f`. That seems like the rare case that you'd just want a 0.0 placeholder instead of a real number.

I guess I can live with the `to_f` check, but it's rarely going to help us catch a bug.

Are there any other places that also look at `respond_to?(:from_now)`?